config/locales/de.yml

```diff
@@ -183,6 +183,10 @@ de:
   client_updated: KlientIn wurde erfolgreich aktualisiert.
   clients: Klienten
   contact_info: Kontakt Informationen
+  contacts:
+    fields:
+      last_name: Nachname
+      name: Name
```

Comment:
with `t_attr(:last_name)` `t_attr(:name)` in the views this whole `contacts: ...` is redundant.